### PR TITLE
Dev101

### PR DESCRIPTION
**Pull request by megassoft**
this is final

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <html lang="en">
-<!-- Main 1 -->
+<!-- Dev 1 -->
```
Comment (kayacetinsu) [inline index.html]: we will revise it

Comment (kayacetinsu) [reply index.html]: ok

**Review by kayacetinsu — COMMENTED** [review]
sure

File: index.html
Comment (megassoft) [inline index.html:1]: literally i added dev 2

**Review by megassoft — COMMENTED** [review]
what can i do?

File: index.html
Comment (kayacetinsu) [reply index.html:1]: try to be specific

Comment (kayacetinsu) [reply index.html:1]: lets see

File: index.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <html lang="en">
-<!-- Main 1 -->
+<!-- Dev 2 -->
```
Comment (megassoft) [inline index.html:3]: is it okay?

Comment (megassoft) [inline index.html:3]: is that okay?

File: index.html [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <html lang="en">
-<!-- Main 1 -->
```
Comment (megassoft) [inline index.html:3]: this is old one

**Review by megassoft — COMMENTED** [review]
i tried few things. check

File: index.html (hunk 2 above)
Comment (kayacetinsu) [reply index.html:3]: yes

Comment (kayacetinsu) [reply index.html:3]: not sure

**Review by kayacetinsu — COMMENTED** [review]
try again

File: index.html (hunk 2 above)
Comment (megassoft) [reply index.html:3]: suggestion?

**Review by megassoft — COMMENTED** [review]
hmm

File: index.html (hunk 2 above)
Comment (kayacetinsu) [reply index.html:3]: we will add ver

Comment (megassoft) [reply index.html:3]: try?

Comment (megassoft) [reply index.html:3]: or

Comment (megassoft) [reply index.html:3]: let me do it

Comment (kayacetinsu) [reply index.html:3]: awesome. done!

Comment (kayacetinsu) [reply index.html:3]: lets publish

Comment (kayacetinsu) [reply index.html:3]: done!

**Review by kayacetinsu — APPROVED** [review]
great! done

**Review by kayacetinsu — APPROVED** [review]
push

**Review by kayacetinsu — CHANGES_REQUESTED** [review]
try

**Review by kayacetinsu — APPROVED** [review]
total

**Review by kayacetinsu — CHANGES_REQUESTED** [review]
again

**Review by kayacetinsu — APPROVED** [review]

**Review by kayacetinsu — CHANGES_REQUESTED** [review]
try

**Review by kayacetinsu — APPROVED** [review]

**Pull request merged**